.github/workflows/main.yml

```diff
@@ -30,15 +30,25 @@ jobs:
       run: dotnet build --no-restore
 
     - name: Run unit tests
-      run: dotnet test --no-build --verbosity normal --filter Laserfiche.Api.Client.UnitTest
+      run: dotnet test tests/unit/Laserfiche.Api.Client.UnitTest.csproj --no-build --verbosity normal --logger "trx;LogFileName=unit-test-results.trx"
```

Comment:
Instead of using test filter to find unit tests, use the csproj path. Using the test filter would create two `trx` files since it would try to run both unit tests and integration tests from this one command, which affects the test reporter.